File: Exporters/IBatchExporter.vb
Imports System.Threading
Imports kCura.WinEDDS.Exporters

Namespace kCura.WinEDDS
	Public Interface IBatchExporter
		Sub Export(artifacts As ObjectExportInfo(), records As Object(), volumePredictions As VolumePredictions(), cancellationToken As CancellationToken)
	End Interface

End Namespace

File: Source/Relativity.Export.Client/Export/VolumeManagerV2/Batches/IBatchExporter.cs
﻿namespace Relativity.Export.VolumeManagerV2.Batches
{
	using System.Threading;

	using kCura.WinEDDS.Exporters;

	public interface IBatchExporter
	{
		void Export(ObjectExportInfo[] artifacts, CancellationToken cancellationToken);
	}
}